test/fixtures/lightscript/if-statement/oneline-return-undelimited/actual.js

```diff
@@ -0,0 +1,2 @@
+f = ->
+  if (true) return 1 else return 2
```

Comment:
Oops I'm an idiot the latter isn't possible; I mean
```
if (true) 1; else 2
```
but that's redundant with the first example so not worth testing